storage/blockblob.go

```diff
@@ -197,6 +197,47 @@ func (b *Blob) PutBlockWithLength(blockID string, size uint64, blob io.Reader, o
 	return b.respondCreation(resp, BlobTypeBlock)
 }
 
+// PutBlockFromURLOptions includes the options for a put block from URL operation
+type PutBlockFromURLOptions struct {
+	PutBlockOptions
+
+	SourceContentMD5   string `header:"x-ms-source-content-md5"`
+	SourceContentCRC64 string `header:"x-ms-source-content-crc64"`
```

Comment:
it might be worth a note that [this is only supported in API version `2019-02-02`](https://docs.microsoft.com/en-gb/rest/api/storageservices/versioning-for-the-azure-storage-services#version-2019-02-02), rather than in the default API version the SDK ships with?

Reply:
Agreed.  In fact we might want to omit this value for earlier API versions unless we're confident it won't cause any issues, i.e. will earlier API versions reject the request due to this value?  It will be used in the signing algorithm.  @ArcturusZhang can you please follow up on this?